t/api/02-iso.t

```diff
@@ -27,7 +27,7 @@ use Test::Output qw/stdout_like stderr_like/;
 use OpenQA::Test::Case;
 use OpenQA::Client;
 use Mojo::IOLoop;
-use Data::Dump;
+use Data::Dumper;
```

Comment:
To be consequent then, I suggest to also remove other unneeded 'use's:

`diff --git a/t/api/02-iso.t b/t/api/02-iso.t
index 0863907..2a0bac6 100644
--- a/t/api/02-iso.t
+++ b/t/api/02-iso.t
@@ -19,15 +19,12 @@ BEGIN {
     unshift @INC, 'lib';
 }
 
-use Mojo::Base -strict;
 use Test::More;
 use Test::Mojo;
 use Test::Warnings ':all';
-use Test::Output qw/stdout_like stderr_like/;
 use OpenQA::Test::Case;
 use OpenQA::Client;
 use Mojo::IOLoop;
-use Data::Dumper;
 
 use OpenQA::IPC;
 use OpenQA::WebSockets;`